DeeployTest/test_platforms.py

```
@@ -987,3 +992,104 @@ def test_gap9_tiled_models_l3_doublebuffer(test_params, deeploy_test_dir, toolch
+@pytest.mark.gap9_w_ne16_tiled
+@pytest.mark.kernels
+@pytest.mark.singlebuffer
+@pytest.mark.l2
+@pytest.mark.parametrize(
+    "test_params",
+    generate_test_params(GAP9_NE16_L2_SINGLEBUFFER_KERNELS, "L2-singlebuffer"),
+    ids = param_id,
+)
+def test_gap9_w_ne16_tiled_kernels_l2_singlebuffer(test_params, deeploy_test_dir, toolchain, toolchain_dir, cmake_args,
+                                                   skipgen, skipsim) -> None:
+    test_name, l1, config_name = test_params
```

Ruff (0.15.10)

[warning] Unpacked variable `config_name` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)